ui/DesignSystem/Component/TrackToggle.svelte

```diff
@@ -6,41 +6,147 @@
 
   export let style = null;
   export let peerCount = null;
+  let hover = false;
+  let active = false;
+  let tracking = false;
```

Comment:
This should be a prop that we can read and write from outside of the component.
```suggestion
  export let tracking = false;
```

Reply:
updated